Sources/Prototyper/Model/APIHandler.swift

```diff
@@ -101,8 +101,9 @@ private let defaultBoundary = "------VohpleBoundary4QuqLuM1cE5lMwCy"
 class APIHandler: ObservableObject {
     var settings: PrototyperSettings
     
+    @EnvironmentObject var state: PrototyperState
```

Comment:
Logging in currently crashes the example application. As far as I can see you can only use an EnvironmentModel in a View and not in a ObservableObject. You will need to either incorporate the functionality of the API Handler in the PrototyperState or maybe the PrototyperState delegates the functionality to the APIHandler.